.github/actions/composite/setupGitForOSBotify/action.yml

```diff
@@ -29,3 +35,11 @@ runs:
       shell: bash
       if: runner.debug == '1'
       run: echo "GIT_TRACE=true" >> "$GITHUB_ENV"
+
+    - name: Generate a token
+      id: generateToken
+      uses: tibdex/github-app-token@021a2405c7f990db57f5eae5397423dcc554159c
+      with:
+        app_id: ${{ inputs.OS_BOTIFY_APP_ID }}
```

Comment:
I think you need to declare these inputs at the top of the file as well. Right now it only has `GPG_PASSPHRASE`